src/main/java/com/shaft/tools/tms/XrayIntegrationManager.java

```diff
@@ -0,0 +1,111 @@
+package com.shaft.tools.tms;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.shaft.tools.io.ReportManager;
+import com.shaft.tools.io.ReportManagerHelper;
+import io.restassured.config.RestAssuredConfig;
+import io.restassured.config.SSLConfig;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Base64;
+import java.util.Calendar;
+
+import static io.restassured.RestAssured.*;
+import static io.restassured.config.EncoderConfig.encoderConfig;
+
+public class XrayIntegrationManager {
+     public static String _JiraAuthorization=
+             Base64.getEncoder().encodeToString(System.getProperty("authorization").getBytes());
+     public static String _ProjectKey= System.getProperty("projectKey");
+     public static String _TestExecutionID=null;
+
+     public static void setup() {
+         baseURI = System.getProperty("jiraUrl");
+         given()
+                 .config(RestAssuredConfig.config().sslConfig(SSLConfig.sslConfig().allowAllHostnames()))
+                 .relaxedHTTPSValidation();
+     }
+
+    public static void importCucumberResults(String filepath) throws Exception {
+
+        setup();
+        String reportPath = System.getProperty("user.dir")+"\\"+filepath;
+        ReportManager.logDiscrete("uploading file: "+reportPath);
+        ReportManager.logDiscrete("Length: "+new File(reportPath).length());
+
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        JsonElement je = JsonParser.parseString( new String(Files.readAllBytes(Paths.get(reportPath))));
+        String prettyJsonString = gson.toJson(je);
+        ReportManager.logDiscrete("Pretty: \n"+ prettyJsonString);
```

Comment:
No need to log the prettified string.
You can actually wrap it by a condition... If SHAFT is running in debug mode to print it for you...
debugMode is a property that takes a boolean value.